tests/fis/test_fis_actions.py

```diff
@@ -0,0 +1,79 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+from chaoslib.exceptions import FailedActivity
+
+from chaosaws.fis.actions import start_experiment
+
+
+def test_that_fis_module_exposes_correct_all_attributes():
```

Comment:
Would test_that_fis_module_correctly_exposes_all_attributes() be better?

Reply:
Hmmm. Perhaps:

```python
def test_that_fis_modules___all___attribute_exposed_correctly():
```

Fits better?

As it's the:

```python
__all__ = ["thing1", "thing2"]
```

Variable it's looking at

Reply:
Yep, perfect 👍

Reply:
Done 😄